k-distribution/tests/regression-new/map-symbolic-tests-haskell/assignment-17-spec.k

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 K Team. All Rights Reserved.
+// generated by gen-tests.sh
+
+requires "map-tests.k"
+
+module ASSIGNMENT-17-SPEC
+    imports MAP-TESTS
+
+    claim
+        <k>
+            assignment ( (MAP:Map X:MyId |-> 1) [ Y:MyId <- 2 ] [ Z:MyId <- 3 ] ) =>
+                assignmentResult ( MAP Z |-> 3 )
+        </k> requires Y ==K X andBool Z  ==K Y andBool Z  ==K X
```

Comment:
This is redundant. Did `gen-tests.sh` generate the spec this way?

```suggestion
        </k> requires Y ==K X andBool Z  ==K Y
```

Reply:
@ttuegel Yes. https://github.com/kframework/k/blob/master/k-distribution/tests/regression-new/map-symbolic-tests/assignment-17-spec.k#L9
The only thing I did with this tests was taking the part from the `.out` file and put it as the rhs.